README.md

```diff
@@ -1,2 +1,118 @@
 # im-identity-api-bundle
-Package for interacting with the Identity API
+Package for interacting with the Identity APIs
+
+## Installation
+This bundle needs to be added to your `composer.json` files as follows
+
+```json
+"repositories": [
+    "immediate/identity-api-bundle": {
+        "type": "vcs",
+        "url": "https://github.com/immediate-media/im-identity-api-bundle.git"
+    },
+]
+```
+
+It will also need to be added as a dependency
+
+```json
+"require": {
+  "immediate/im-identity-api-bundle": "^1.0",
+}
+```
+
+The below should automatically be added to your `config/bundles.php` file as part of a `composer update`. 
+
+```php
+IM\Fabric\Bundle\IdentityApiBundle\IdentityApiBundle::class => ['all' => true]
+```
+
+If not, run `composer recipes:install immediate/im-identity-api-bundle --force`
+
+Alongside the package installation, you will also need to require packages that include implementations for PSR 7, PSR 17 and PSR 18 interfaces.
+
+## Configuration
```

Comment:
Can we update this to use `Configuration: Symfony` & `Configuration: WordPress` sections please (assuming from the wp_mock dependency it supports both.

Reply:
It does not. I'm removing the `wp_mock` for now.